specification/azuredata/resource-manager/Microsoft.AzureData/preview/2017-03-01-preview/azuredata.json

```diff
@@ -0,0 +1,873 @@
+        },
+        "name": {
+          "description": "Resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Resource type.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "TrackedResource": {
+      "description": "ARM tracked top level resource.",
+      "required": [
+        "location"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "properties": {
+        "location": {
+          "description": "Resource location.",
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SqlServerRegistration": {
+      "description": "A SQL server registration.",
+      "required": [
+        "location"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SqlServerRegistrationProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "SqlServerRegistrationUpdate": {
+      "description": "An update to a SQL Server Registration.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "SqlServerRegistrationListResult": {
+      "description": "Server.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SqlServerRegistration"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "SqlServerProperties": {
+      "description": "The SQL server properties.",
+      "type": "object",
+      "properties": {
+        "cores": {
+          "format": "int32",
+          "description": "Cores of the Sql Server.",
+          "type": "integer",
+          "x-ms-mutability": [
+            "update"
+          ]
+        },
+        "version": {
+          "description": "Version of the Sql Server.",
+          "type": "string"
+        },
+        "edition": {
+          "description": "Sql Server Edition.",
+          "type": "string"
+        },
+        "registrationID": {
```

Comment:
should we use registration**Id**?

Reply:
this was already in public preview, may be we can leave it as is